index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
The directions want you to add 2 `meta` tags here. Some examples you can add are at this link: [https://www.w3schools.com/tags/tag_meta.asp](https://www.w3schools.com/tags/tag_meta.asp).